app/assets/stylesheets/layout.scss

```diff
@@ -246,6 +246,10 @@ table tbody td[align="right"], table tbody th[align="right"], table thead th[ali
   word-break: break-all;
 }
 
+.live-red-text {
```

Comment:
We've added similar, including font-weight.  We should probably use the same, e.g.
.live-red-text {
  color: red;
  font-weight: 600;
}

Reply:
Have added in the same. Once we get to the point of merging various branches we can probably just use the actual same style class.